source/sass/component/_select.scss

```diff
@@ -29,16 +26,15 @@ $c-select-border-radius-option: var(--c-select-border-radius-option, $border-rad
     margin: 0;
     outline: 0;
 
-    height: calc(#{$base}*6);
-    padding: map-get($spacers, "2");
+    height: map-get($spacers, "6");
+    padding: 0 map-get($spacers, "2");
+
     font-size: calc(#{$base}*2);
```

Comment:
while we're cleaning up wouldn't it be neat to create a variable for this in `source/sass/setting/_var_fonts.scss`?
something like `$font-size-select` and then we could also utilize that for the `.c-select__icon`'s `line-height`? seems like that would makes things easy to make an consistent